api/V1/Visits.ts

```diff
@@ -50,19 +56,23 @@ function getTrackTag(trackTags: TrackTag[], userID: number): TrackTag | null {
   }
   const manualTags = trackTags.filter((tag) => tag.automatic == false);
   if (manualTags.length > 0) {
-    const userTag = manualTags.find((tag) => tag.UserId == userID);
-    if (userTag) {
-      return userTag;
-    } else {
-      return manualTags[0];
+    const animalTags = manualTags.filter(
+      (tag) => !nonAnimalTags.includes(tag.what)
+    );
+    const uniqueTags = new Set(animalTags.map((tag) => tag.what));
+    if (uniqueTags.size > 1) {
+      const conflict = {
+        what: conflictTag,
+        confidence: manualTags[0].confidence,
+        automatic: false
+      };
+      return conflict as TrackTag;
     }
+
+    return animalTags.length == 0 ? manualTags[0] : animalTags[0];
```

Comment:
Still don't want to return a meta tag or do you filter these out later.

Reply:
whoops, thought i added that